Formula/xapian.rb

```diff
@@ -39,6 +28,9 @@ class Xapian < Formula
   deprecated_option "ruby" => "with-ruby"
 
   depends_on :python => :optional
+  depends_on "sphinx-doc" if build.with? "python" => :build
```

Comment:
There's currently no way to build the bindings without documentation (for those bindings; it's totally different to the documentation for Xapian itself, which doesn't get built by this formula), so you'll get a build error if you try.